report_async/static/src/xml/report_async.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name='ReportAsyncConfiguration'>
+        <div class="form">
+            <div class="form-group">
+                <label for="async-user-email">Email Address</label>
+                <input type="email" class="form-control" id="async-user-email"
+                       aria-describedby="emailHelp"
+                       placeholder="admin@example.com"/>
+                <small id="async-user-email-help" class="form-text text-muted">
+                    Email will be used to send the report after queue job is
+                    done on the background
+                </small>
+            </div>
+            <div class="form-group">
+                <div class="form-check">
+                    <input type="checkbox" class="form-check-input"
+                           id="async_report_checker"/>
+                    <label class="form-check-label" for="async_report_checker">
+                        Async Report
+                    </label>
+                </div>
```

Comment:
Feels more logical to have this field at the top and checked by default. Then the email address comes below it.